README.md

```diff
@@ -1,15 +1,23 @@
 # Uphold Validator.js
 
-Extensive validations built on top of validator.js.
+Extensive validations built on top of
+[validator.js](https://github.com/guillaumepotier/validator.js).
 
-Improvements over validator.js:
- - extended set of asserts
- - logging with debugnyan
- - throwing specific errors on validation failures
+Improvements over `validator.js`:
+
+- [extended set of asserts](https://github.com/uphold/validator.js-asserts)
+- logging with [debugnyan](https://github.com/uphold/debugnyan)
+- throwing specific errors on validation failures
```

Comment:
I'm not really sure what this refers to exactly. It would be nice to add a link for additional context here as well.

For reference, I thought it could have something to do with [validator.js-validate](https://github.com/uphold/validator.js-validate) and/or [validator.js-validate](https://github.com/uphold/validator.js-validate), but there doesn't seem to be any reference to them in this codebase. Any suggestions welcome.